addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -96,7 +96,7 @@ class Applicant(models.Model):
     name = fields.Char("Subject / Application Name", required=True)
     active = fields.Boolean("Active", default=True, help="If the active field is set to false, it will allow you to hide the case without removing it.")
     description = fields.Text("Description")
-    email_from = fields.Char("Email", size=128, help="Applicant email", compute='_compute_partner_phone_email', store=True, readonly=False)
+    email_from = fields.Char("Email", size=128, help="Applicant email", compute='_compute_partner_phone_email', inverse='_inverse_partner_email', store=True, readonly=False)
```

Comment:
Remove the `readonly=False`, computed fields with inverse are `readonly=False` ;).